#!/usr/bin/python3
def raise_exception_msg(message=""):
    """Raise an exceptiom NameError with a message."""
    raise NameError(message)
